CastleBoy/entity.cpp

```diff
@@ -296,16 +296,20 @@ Entity* Entities::add(uint8_t type, int16_t x, int8_t y)
   return NULL;
 }
 
-signed char getMovingPlatformDirection(Entity& entity)
+char getMovingPlatformDirection(Entity& entity)
```

Comment:
Don't use 'char'. The C standard doesn't define if it's a signed type or not. I suggest using the explicit type 'int8_t' instead.